util/unresolved_addr.go

```diff
@@ -21,10 +21,28 @@ import (
 	"net"
 )
 
-// TestAddr is an address to use for test servers. Listening on port 0 causes
-// the kernel to allocate an unused port.
+// TestAddr is an address to use for test servers. Listening on port 0
+// causes the kernel to allocate an unused port.
 var TestAddr = NewUnresolvedAddr("tcp", "127.0.0.1:0")
 
+// IsolatedTestAddr is initialized in testaddr_*.go
+
```

Comment:
No, it's deliberate - the docstring starts on the following line; the preceding comment is for readers of the source but is not a part of the godoc documentation.